plugins/jobs/server/job_rest.py

```diff
@@ -33,6 +33,8 @@ def __init__(self):
         self.route('GET', ('all',), self.listAllJobs)
         self.route('GET', (':id',), self.getJob)
         self.route('PUT', (':id',), self.updateJob)
+        self.route('GET', (':id', 'child',), self.getChildJobs)
+        self.route('PUT', (':id', 'parent',), self.setParentJob)
```

Comment:
I would also expect setParentJob(...) be a single shot thing ( as in real life :smile: ), once you have a parent, that is it!